### DISCUSSION TITLE
TaxExemptionReason e TaxExemptionCode

### DISCUSSION THREAD

**Discussion by MarioEME** [assoft-portugal/SAF-T-AO#145] [Q&A]
Boa tarde,

Estou a tentar perceber que valores preencher no  TaxExemptionReason e TaxExemptionCode mas não encontro informação que me ajude.

Quando o TaxType e TaxCode são NS, quais os valores que devem ser preenchidos no TaxExemptionReason e TaxExemptionCode 

![image](https://user-images.githubusercontent.com/24740118/111669983-3f21d180-880f-11eb-9799-b2ca5a9a534a.png)

Obrigado.

**Comment by cryptolopes** [comment]
#114 

No regime de não sujeição não deveria ser necessário preencher esses elementos, mas, na minha opinião, é:

M02 | Transmissão de bens e serviços não sujeita |
-- | -- |

**Reply by MANUEL2810** [reply]
bom dia, usei a tua sugestão e ainda assim o erro permanece.....
